tests/AutoReview/ProjectCodeTest.php

```diff
@@ -339,6 +339,7 @@ private function getTestClasses()
             ->in(__DIR__.'/..')
             ->exclude(array(
                 'Fixtures',
+                'Test',
```

Comment:
Isn't the point of moving those files from `src` to `tests` to *not* expose them anymore, and thus mark them all as `@internal`?